packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -56,9 +57,16 @@ const secondarySidebarVariants = {
 	mobileOpen: { width: '100vw' },
 };
 
+const secondarySidebarDiaglogVariants = {
+	open: { width: SECONDARY_SIDEBAR_DIALOG_WIDTH },
+	closed: { width: 0 },
+	mobileOpen: { width: '100vw' },
+};
```

Comment:
This assumes that the width of the sidebar changes in zoom-out, that is not always true. The width only changes when you open the pattern categories.